### PR TITLE
use specinfra with freebsd mount(8) fix

### LINKED ISSUE #7: fix linuxism in specinfra
See https://github.com/reallyenglish/ansible-role-logstash/issues/7

### PR DESCRIPTION
**Pull request by ochko**
fixes #7

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,4 +14,4 @@ gem 'kitchen-verifier-shell', '~> 0.2.0'
 gem 'kitchen-verifier-serverspec', '~> 0.3.0'
 gem 'infrataster', '~> 0.3.2', :git => 'https://github.com/trombik/infrataster.git', :branch => 'reallyenglish'
 gem 'serverspec', '~> 2.37.2'
-gem 'specinfra', '>= 2.63.2' # OpenBSD's `port` is fixed in this version
+gem 'specinfra', '>= 2.66.6'
```
Comment (trombik) [inline Gemfile:17]: when you changes files that `qansible` creates by default, and the modification should be applied to all roles, please create an issue in `qansible` repository. `qansible qa` warns when `Gemfile` is not identical with the default `Gemfile`, and this ensures certain, and same, level of quality in all roles.

**Review by trombik — COMMENTED** [review]
the failure in Jenkins is caused by auto-update, which is enabled by default. it should have been fixed in https://github.com/reallyenglish/packer-templates/pull/5

will look into it.

**Comment by trombik** [comment]
well, my knee-jerk analysis was wrong. the platform is debian.
```
packages ...", "Fetched 78.7 MB in 3min 19s (395 kB/s)", "dpkg: error: dpkg status database is locked by another process"]}[0m
       	to retry, use: --limit @/tmp/kitchen/default.retry
       
       PLAY RECAP *********************************************************************
       [0;31mlocalhost[0m                  : [0;32mok=9   [0m changed=0    unreachable=0    [0;31mfailed=1   [0m
       
>>>>>> Converge failed on instance <default-debian-86-amd64>.
```

**Comment by trombik** [comment]
`trombik/ansible-debian-8.7-amd64` version 1.0.1 should fix this

**Comment by ochko** [comment]
@trombik FYI, `bundle exec kitchen test` fails for debian-8.7 with:

```
       downloading role 'apt-repo', owned by -%20name%3A%20reallyenglish
       Sorry, - name: reallyenglish.apt-repo was not found on galaxy.ansible.com.
       Going to invoke ansible-playbook with: ANSIBLE_ROLES_PATH=/tmp/kitchen/roles:/tmp/kitchen/extra_modules:/tmp/kitchen/filter_plugins:/tmp/kitchen/extra_roles sudo -Es  ansible-playbook -i /tmp/kitchen/hosts  -c local -M /tmp/kitchen/modules -v   --vault-password-file=/tmp/kitchen/ansible_vault_key     /tmp/kitchen/default.yml
       ERROR: cannot find role in /tmp/kitchen/roles/reallyenglish.apt-repo or /tmp/kitchen/reallyenglish.apt-repo or /tmp/kitchen/roles/reallyenglish.apt-repo or /tmp/kitchen/extra_modules/reallyenglish.apt-repo or /tmp/kitchen/filter_plugins/reallyenglish.apt-repo or /tmp/kitchen/extra_roles/reallyenglish.apt-repo
>>>>>> Converge failed on instance <default-debian-87-amd64>.
>>>>>> Please see .kitchen/logs/default-debian-87-amd64.log for more details
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
```

BTW, do we use debian?

**Comment by trombik** [comment]
bug fixed version has been released and the commit is being built.

for the reason why supporting multiple platforms is important, see:
https://github.com/reallyenglish/ansible-role-example/wiki/Creating_Role#support-multiple-platforms

you are also enjoying benefits of portability everyday.
https://www.openbsd.org/papers/portability.pdf

**Pull request merged**